Update scheduler-agent-supervisor-content.md (#4449)
MicrosoftDocs/architecture-center: docs/patterns/scheduler-agent-supervisor-content.md

- This pattern can be difficult to implement and requires thorough testing of each possible failure mode of the system.

> - The recovery/retry logic implemented by the Scheduler is complex and dependent on state information held in the state store. It might also be necessary to record the information required to implement a compensating transaction in a durable data store. Note that compensating transactions don't always work.

Reviewer: ```suggestion
- The recovery/retry logic implemented by the Scheduler is complex and dependent on state information held in the state store. It might also be necessary to record the information required to implement a compensating transaction in a durable data store. A compensating transaction might fail as well.
```
